app/src/main/java/org/fossasia/openevent/app/module/event/list/EventsPresenter.java

```diff
@@ -61,6 +79,27 @@ public void loadUserEvents(boolean forceReload) {
             return eventsDataRepository.getEvents(forceReload);
     }
 
+    private void listenToModelChanges() {
+        eventListener.startListening();
+
+        eventListener.getNotifier()
+            .compose(dispose(getDisposable()))
+            .compose(erroneous(getView()))
+            .filter(eventModelChange -> eventModelChange.getAction().equals(BaseModel.Action.UPDATE))
+            .map(DatabaseChangeListener.ModelChange::getModel)
+            .subscribe(event -> updateLocal(event), Logger::logError);
+    }
+
+    private void updateLocal(Event event) {
+        Utils.indexOf(events, event, (first, second) -> first.id == second.id)
+            .subscribeOn(Schedulers.computation())
+            .subscribe(index -> {
+                events.set(index, event);
+                Collections.sort(events);
+                getView().updateEvent(event);
+            }, Logger::logError);
+    }
```

Comment:
This isn't required, because this screen will never be opened when the user toggles the state of an event